src/Controls/src/Core/Layout/GridLayout.cs

```diff
@@ -228,6 +228,39 @@ public override void Add(IView child)
 			}
 		}
 
+		public void Add(IView view, int left, int top)
```

Comment:
@hartez I added this to make the code run, but we probably want to IGridChildList or something to add the AddHorizontal.